lib/src/soloud.dart

```diff
@@ -1232,6 +1232,53 @@ interface class SoLoud {
     SoLoudController().soLoudFFI.setVolume(handle, volume);
   }
 
+  /// Get a sound's current pan setting.
+  ///
+  /// [handle] the sound handle.
+  /// Returns the range of the pan values is -1 to 1, where -1 is left, 0 is
+  /// middle and and 1 is right.
+  ///
+  /// Throws [SoLoudNotInitializedException] if the engine is not initialized.
+  double getPan(SoundHandle handle) {
+    if (!isInitialized) {
+      throw const SoLoudNotInitializedException();
+    }
+    return SoLoudController().soLoudFFI.getPan(handle.id);
+  }
+
+  /// Set a sound's current pan setting.
+  ///
+  /// [handle] the sound handle.
+  /// [pan] the range of the pan values is -1 to 1, where -1 is left, 0 is
+  /// middle and and 1 is right.
+  ///
+  /// Throws [SoLoudNotInitializedException] if the engine is not initialized.
+  void setPan(SoundHandle handle, double pan) {
```

Comment:
What happens if the user tries to set pan to `123.0` or `double.infinity`? Should we `assert(-1 <= pan && pan <= 1)` and then `.clamp()` the value?